blueprint/index.md

```diff
@@ -1,224 +1,265 @@
 ---
-title: Microsoft Teams meeting on Genesys Cloud Blueprint
+title: Set up Microsoft Teams meeting on Genesys Cloud
+author: {author name}
```

Comment:
This isn't valid and broke the build. Please ensure actual values are used in the future.